src/patternfly/demos/Masthead/templates/masthead-demo--drilldown.hbs

```diff
@@ -0,0 +1,143 @@
+            {{/menu-header}}
+            {{#> menu-search}}
+              {{#> menu-search-input}}
+                {{#> search-input search-input--placeholder="Search"}}{{/search-input}}
+              {{/menu-search-input}}
+            {{/menu-search}}
+            {{> divider}}
+            {{#> menu-group}}
+              {{#> menu-group-title}}
+                Favorites
+              {{/menu-group-title}}
+              {{#> menu-list}}
+                {{#> menu-list-item newcontext}}
+                  {{#> menu-item menu-item--IsLink="true"}}
+                    {{#> menu-item-main}}
+                      {{#> menu-item-text}}
+                        Link 1
+                      {{/menu-item-text}}
+                    {{/menu-item-main}}
+                  {{/menu-item}}
+                  {{> menu-item-action menu-item-action--IsFavorite="true" menu-item-action--IsFavorited="true"}}
+                {{/menu-list-item}}
+                {{#> menu-list-item newcontext}}
+                  {{#> menu-item menu-item--IsLink="true" menu-item--attribute='target="_blank"'}}
+                    {{#> menu-item-main}}
+                      {{#> menu-item-text}}
+                        Link 2
+                      {{/menu-item-text}}
+                      {{> menu-item-external-icon}}
+                    {{/menu-item-main}}
+                  {{/menu-item}}
+                  {{> menu-item-action menu-item-action--IsFavorite="true"}}
+                {{/menu-list-item}}
+              {{/menu-list}}
+            {{/menu-group}}
+            {{> divider}}
+            {{#> menu-group}}
+              {{#> menu-group-title}}
+                Group 1
+              {{/menu-group-title}}
+              {{#> menu-list}}
+                {{#> menu-list-item newcontext}}
+                  {{#> menu-item menu-item--IsLink="true"}}
+                    {{#> menu-item-main}}
+                      {{#> menu-item-text}}
+                        Link 1
+                      {{/menu-item-text}}
+                    {{/menu-item-main}}
+                  {{/menu-item}}
+                  {{> menu-item-action menu-item-action--IsFavorite="true"}}
+                {{/menu-list-item}}
+                {{#> menu-list-item newcontext}}
+                  {{#> menu-item menu-item--IsLink="true" menu-item--attribute='target="_blank"'}}
+                    {{#> menu-item-main}}
+                      {{#> menu-item-text}}
+                        Link 2
+                      {{/menu-item-text}}
+                      {{> menu-item-external-icon}}
+                    {{/menu-item-main}}
+                  {{/menu-item}}
+                  {{> menu-item-action menu-item-action--IsFavorite="true" menu-item-action--IsFavorited="true"}}
+                {{/menu-list-item}}
+              {{/menu-list}}
+            {{/menu-group}}
+          {{/menu}}
+        {{/menu-list-item}}
+      {{/menu-list}}
+    {{/menu-group}}
+  {{/menu-content}}
+{{/menu}}
```

Comment:
For the mobile example in the masthead demos, the search doesn't seem to span the whole width of the "white block". Not sure what that extra white block on the right is, but would make the search span the width of it, or make the extra white blow black? Please let me know if that made no sense haha

<img width="496" alt="image" src="https://user-images.githubusercontent.com/51165119/154133400-f56a6cad-a29e-4b4b-92cd-afec9f240d53.png">